modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,77 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates of data
+from the server to the client.
+To do this GraphQL Subscriptions utilise websockets to enable clients to subscribe to published changes.
+
+The architecture of websockets is as follows:
+
+* Client connects to websocket server
+* Upon certain events, the server can publish the results of these events to the websocket
+* Any _currently connected_ client to that websocket will receive these results
+* The client can close the connection at any time and will no longer recieve updates
+
+Websockets are not however, a queueing system. For example if the client is offline when the server publish the changes
```

Comment:
I always prefer positive messages rather than negation. So instead of websockets are not.. we can say: "Websockets are perfect solution for delivering messages to currently active clients". Then we can follow up - for sending updates to non active clients we recommend push notifications. Mentioning queue here is not going to increase understanding as users can actually implement queue with changesets if they want to. So what we explain here is that we took upstream solution. Tested it with our features. Added auth and provide reference implementation if you looking for inspiration.

This is more advice rather than pointing some problem.